noir/noir-repo/aztec_macros/src/transforms/compute_note_hash_and_optionally_a_nullifier.rs

```diff
@@ -69,14 +69,14 @@ pub fn inject_compute_note_hash_and_nullifier(
         let max_note_length_const = get_global_numberic_const(context, "MAX_NOTE_FIELDS_LENGTH")
             .map_err(|err| {
                 (
-                    AztecMacroError::CouldNotImplementComputeNoteHashAndNullifier {
+                    AztecMacroError::CouldNotImplementcomputeNoteHashAndOptionallyANullifier {
```

Comment:
Error name is not in camelCase

Reply:
Thanks, fixed in https://github.com/AztecProtocol/aztec-packages/pull/6948/commits/896e70d72265d3c1d733058686fdb06f8c895b0c